src/InstrumentationEngine.Attach/CommandHandler/AttachCommandHandler.cs

```diff
@@ -109,7 +109,8 @@ internal class AttachCommandHandler
                     readerSettings.ValidationType = ValidationType.Schema;
 
                     // Read configuration source file
-                    using (XmlReader reader = XmlReader.Create(sourceInfo.ConfigSourceFilePath, readerSettings))
+                    using (var stream = new StreamReader(sourceInfo.ConfigSourceFilePath))
```

Comment:
Can you add comments here as to why this is being done so that someone doesn't come along later and revert the behavior?